include/tst_clone.h

```diff
@@ -34,6 +34,6 @@ int ltp_clone_malloc(unsigned long clone_flags, int (*fn)(void *arg),
 int ltp_clone_quick(unsigned long clone_flags, int (*fn)(void *arg),
 		void *arg);
 
-#define clone(...) (use_the_ltp_clone_functions, do_not_use_clone)
+#define clone(...) (use_the_ltp_clone_functions__do_not_use_clone)
 
```

Comment:
The NDK's cross compiler, which looks like a patched gcc 4.9.x, craps out with:
```
/home/sj/work/android-ltp/toolchain-r13b/bin/arm-linux-androideabi-gcc -static -I/home/sj/work/android-ltp/toolchain-r13b/sysroot/usr/include -I/home/sj/work/android-ltp/toolchain-r13b/usr/include -g -O2 -fno-strict-aliasing -pipe -Wall -W -Wold-style-definition -Werror-implicit-function-declaration  -I/home/sj/work/android-ltp/ltp/testcases/kernel/include -I../../../../include -I../../../../include -I../../../../include/old/ -static -L/home/sj/work/android-ltp/toolchain-r13b/usr/lib  -L../../../../lib -L../../../../lib/android_libpthread -L../../../../lib/android_librt  ppoll01.c   -lltp -o ppoll01
In file included from ../../../../include/tst_test.h:35:0,
                 from ppoll01.c:32:
../../../../include/tst_clone.h:37:48: error: expected ')' before ',' token
 #define clone(...) (use_the_ltp_clone_functions, do_not_use_clone)
                                                ^
```
Would you prefer me to investigate the root cause or is a separate commit with a comment saying Android's NDK doesn't like it enough?